src/component/endorsements/endorsement.jsx

```diff
@@ -0,0 +1,197 @@
+import React, { useState, useEffect } from "react";
+import {
+  Card,
+  Typography,
+  CardActions,
+  IconButton,
+  Snackbar,
+  CardContent,
+  Chip
+} from "@material-ui/core";
+import AddCircleIcon from "@material-ui/icons/AddCircle";
+import MuiAlert from "@material-ui/lab/Alert";
+import AddIcon from "@material-ui/icons/Add";
+
+import validate from "./yup";
+
+import { useStyles } from "./style";
+
+import { Modal } from "./features";
+
+const Endorsement = () => {
+  const classes = useStyles();
+  const [open, setOpen] = useState(false);
+  const [openAlert, setOpenAlert] = React.useState(false);
+  const [value, setValue] = useState("");
+  const [endorsement, setEndorsement] = useState([]);
+
+  const handleAlert = () => {
+    setOpenAlert(true);
+  };
+
+  const handleAlertClose = () => {
+    setOpenAlert(false);
+  };
```

Comment:
1. unnecessary use of extra functions setOpenAleart is in itself a function and can be used to set the state of  openAlert true or false
2. handleAleart and handleAlertClose could have been single functions as the perform the same role